doc/doxygen/headers/coding_conventions.h

```diff
@@ -60,6 +60,10 @@ style guidelines outlined in this page.</p>
   functions which clear bits of flags should be named <code>clear_*</code>.
   Example: CellIterator::set_refine_flag().</li>
 
+<li> Punctuation operators should be used instead of their English
+  equivalents (i.e., use <code>&&</code>, <code>||</code>, and <code>!</code>
+  instead of <code>and</code>, <code>or</code>, and <code>not</code>).
```

Comment:
Is "punctuation operators" how they're called? Not "logical operator symbols" or similar? (I don't think I've ever heard the term you use, but that of course doesn't mean that it's wrong.